File: scholar-business/src/main/java/com/varun/scholar/business/data/entities1/PerishableSearch.java
package com.varun.scholar.business.data.entities1;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import com.varun.scholar.business.data.entities.AbstractEntity;

@Entity
public class PerishableSearch extends AbstractEntity {
	String itemId;
	String itemName;
	String itemDescription;
	String quantity;
	@Temporal(TemporalType.DATE)
	Date itemAcquisitionDate;
}

File: scholar-web/src/main/webapp/app/model/administration/inventory/perishable/PerishableSearch.js
Ext.define('scholar.model.administration.inventory.perishable.PerishableSearch', {
	extend: 'Ext.data.Model',		
	fields : [ {
		name : 'id',
		type : 'int'
	},{
		name : 'itemId',
		type : 'string'
	}, {
		name : 'itemName',
		type : 'string'
	},
	{
		name : 'itemDescription',
		type : 'string'
	},
	{
		name : 'quantity',
		type : 'string'
	},
	{
		name : 'itemAcquisitionDate',
		type : 'date',
		dateFormat : 'n/j h:ia'
	}  ]
});